src/python_testing/TC_CC_2_2.py

```diff
@@ -48,62 +48,80 @@ def default_timeout(self) -> int:
     def steps_TC_CC_2_2(self):
         THcommand = "Test Harness sends the"
 
-        def store_values(attr: str) -> str:
-            return f"TH stores the reported values of _{attr}_ in all incoming reports for _{attr}_ attribute, that contains data in _reportedCurrentHueValuesList_, over a period of 20 seconds."
+        def store_values(reportList: str, attr: str) -> str:
+            return f"TH stores the reported values of _{attr}_ in all incoming reports for _{attr}_ attribute, that contains data in _{reportList}_, over a period of 20 seconds."
 
-        def verify_entry_count(attr: str) -> str:
-            return f'TH verifies that _reportedCurrentHueValuesList_ does not contain more than 10 entries for _{attr}_'
+        def verify_entry_count(reportList: str, attr: str) -> str:
+            return f'TH verifies that _{reportList}_ does not contain more than 12 entries for _{attr}_'
```

Comment:
All the {} will break Python string handling at runtime. Remove the `f` prefix

Reply:
Not sureI understand, for this specifically the f was already there due to the "attr" string, I only added which list it is stored in so match TP?

Reply:
Please avoid using things like `{THCommand}` in the future, and just write it out.

Reply:
Oh you are thinking about these, sure. We have likely not landed on the perfect solution for this but it is to accommodate the use of document attributes in the test plan and I think @cecille did some stuff to address this on the python side as well, but not sure. But that is not really a new thing that I added in this PR though 🤷🏻‍♂️